test/src/e2e.dynval/dv.changeParams.test.ts

```diff
@@ -202,4 +202,101 @@ describe("Change commonParams", function() {
             await checkValidators(checkingNode.sdk, expectedValidators);
         });
     });
+    describe("Change the maximum size of candidate metadata", async function() {
+        function nominationWithMetadata(size: number) {
+            return stake.createSelfNominateTransaction(
+                nodes[0].sdk,
+                1,
+                " ".repeat(size)
+            );
+        }
+
+        it("Should apply larger metadata limit after increment", async function() {
+            const termSeconds = 20;
+            const margin = 1.2;
+
+            this.slow(termSeconds * margin * 1000);
+            this.timeout(termSeconds * 2 * 1000);
+
+            const [alice] = allDynValidators;
+            const checkingNode = nodes[0];
+            const changeTxHash = await changeParams(checkingNode, 1, {
+                ...defaultParams,
+                maxCandidateMetadataSize: 256
```

Comment:
I think it would be nice to give `maxCandidateMetadataSize` as a parameter directly to the function `withNodes` as 128. The default value is 128, but I hope the test will work even if the default changes later.

Reply:
How do you think about using `defaultParams.maxCandidateMetadataSize * 2` or similar instead of constant value?

Reply:
I think adding `maxCandidateMetadataSize` to `withNodes` is good too. I'll fix it this way.